x/auth/types/test_utils.go

```diff
@@ -7,71 +7,10 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func NewTestStdFee() StdFee {
-	return NewStdFee(100000,
-		sdk.NewCoins(sdk.NewInt64Coin("atom", 150)),
-	)
-}
-
-// coins to more than cover the fee
-func NewTestCoins() sdk.Coins {
-	return sdk.Coins{
-		sdk.NewInt64Coin("atom", 10000000),
-	}
-}
-
+// KeyTestPubAddr generates a new secp256k1 keypair.
 func KeyTestPubAddr() (crypto.PrivKey, crypto.PubKey, sdk.AccAddress) {
```

Comment:
Related to the above comment, I'm inclined to put this function into `testdata/`, so that we can remove this file.

wdyt?

Reply:
Either that or `/testutil`